test/ctr.bats

```diff
@@ -265,6 +265,9 @@ function wait_until_exit() {
 	run crictl inspect "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
+	run crictl inspect "$ctr_id" | jq -e ".info.privileged == false"
```

Comment:
This check is not working because

1. in `run cmd1 ... | cmd2` the part starting with the pipe
       character is not part of `run` statement;
    
2. `run` eats `cmd1 ...` output (into `$output` variable);
    
so `cmd2` is provided with empty input. Now,
    
3. `jq` with empty input does not run any filters and thus succeeds
  (even with `-e`, see https://github.com/stedolan/jq/issues/1628)

Reply:
Sigh, yep that's right. Is there a fix underway or should I work on it?